.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
+concurrency = eventlet
+source = nameko
```

Comment:
I think we should if possible. Will need a bit of work to get it up to 100% though. Separate PR I think